### DISCUSSION TITLE
[Question] - TreeView: Cannot select nodes that have children

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/ark#2981] [General]
### Description

https://github.com/user-attachments/assets/a7e165bd-42bc-4328-81b5-e0273d0d86a0

As seen on the video above, I am not sure how we can make a node with children selectable.

### Link to Reproduction (or Detailed Explanation)

None

### Steps to Reproduce

None

### Ark UI Version

4.1.0

### Framework

- [X] React
- [ ] Solid
- [ ] Vue

### Browser

Chrome

**Comment by xeinebiu** [comment]
@segunadebayo any idea on this case ?

**Comment by segunadebayo** [comment]
Closing — TreeView supports selecting nodes with children. Feel free to reopen if still experiencing issues.
